### DISCUSSION TITLE
404 node-v111-linux-x64-Release.tar.gz

### DISCUSSION THREAD

**Discussion by noahrama** [maplibre/maplibre-native#2496] [Q&A]
**Describe the bug**
this link is dead: https://github.com/maplibre/maplibre-gl-native/releases/download/node-v5.2.0/node-v111-linux-x64-Release.tar.gz

```
node-pre-gyp http GET https://github.com/maplibre/maplibre-gl-native/releases/download/node-v5.2.0/node-v111-linux-x64-Release.tar.gz
node-pre-gyp ERR! install response status 404 Not Found on https://github.com/maplibre/maplibre-gl-native/releases/download/node-v5.2.0/node-v111-linux-x64-Release.tar.gz

```

**Comment by acalcutt** [comment]
We are only supporting stable releases of node (14,16,18) right now, see https://github.com/maplibre/maplibre-gl-native/tree/main/platform/node

we aren't generating binaries for node 19 (abi 111) , which is why there is not a binary for node-v111 . You can see at https://github.com/maplibre/maplibre-gl-native/releases/tag/node-v5.2.0 that file doesn't exists

**Comment by acalcutt** [comment]
There were changes in the last release that allows maplibre-native to build for node 19, but it is currently not in the list of versions we are publishing ( https://github.com/maplibre/maplibre-gl-native/blob/main/platform/node/scripts/publish.sh#L12 )
